src/index.js

```diff
@@ -1,45 +1,116 @@
+/* eslint-disable no-use-before-define */
 import './styles.css';
 
-const tasksArray = [
-  {
-    description: 'Wash the dishes',
-    completed: false,
-    index: 3,
-  },
-  {
-    description: 'Complete To Do List project',
-    completed: false,
-    index: 2,
-  },
-  {
-    description: 'Complete Microverse exercises',
-    completed: false,
-    index: 1,
-  },
-];
-
-tasksArray.sort((a, b) => a.index - b.index);
-
-tasksArray.forEach((task) => {
-  const element = document.querySelector('tbody');
-  const tr2 = document.createElement('tr');
-  const td2 = document.createElement('td');
-  const input = document.createElement('input');
-  input.type = 'checkbox';
-  td2.appendChild(input);
-  tr2.appendChild(td2);
-  const td3 = document.createElement('td');
-  td3.classList.add('align');
-  const label = document.createElement('label');
-  label.textContent = task.description;
-  td3.appendChild(label);
-  const td4 = document.createElement('td');
-  td4.innerHTML = `<i class='fa-solid fa-ellipsis-vertical'>
-  </i>`;
-  tr2.appendChild(td3);
-  tr2.appendChild(td4);
-  element.appendChild(tr2);
-  const table = document.querySelector('table');
-  table.appendChild(element);
-  document.body.appendChild(table);
-});
\ No newline at end of file
+let tasksArray = [];
+
+function addTask() {
+  const input = document.querySelector('#task');
+  const value = input.value.trim();
+  if (value !== '') {
+    const task = {
+      description: value,
+      completed: false,
+      index: tasksArray.length + 1,
+    };
+    tasksArray.push(task);
+    input.value = '';
+    addtoLocalStorage();
+  }
+  displayTasks();
+}
+
+function updateTaskIndexes() {
+  tasksArray.forEach((task, index) => {
+    task.index = index + 1;
+  });
+}
```

Comment:
- [ ] The requirements say that you should create a new file for the new functionality 😄 , You did a great work from here , now I will suggest you to create the new file for the `add` , `delete` and `update` feature 👍

Reply:
This has been fixed, sorted